src/main/java/com/kainos/ea/resources/WebService.java

```diff
@@ -273,13 +272,18 @@ public Response login(User user) throws SQLException, IOException, ClassNotFound
         }
     }
 
+    /**
+     * Edit a job from database
+     * @param job The job you wish to edit.
+     * @return Status 200. OK if editing succeeds. Status 500. Internal Server Error otherwise.
+     * @throws SQLException
```

Comment:
Can you add a message next to sql exception and the other exception

Reply:
Done